app/scripts/modules/google/src/serverGroup/configure/serverGroupCommandBuilder.service.js

```diff
@@ -422,6 +431,9 @@ module.exports = angular
           tags: [],
           labels: {},
           availabilityZones: [],
+          enableSecureBoot: serverGroup.enableSecureBoot,
```

Comment:
Looks OK for now, but if we expand the feature set here it might be worth encapsulating this set of fields.

Reply:
Agreed.